### PR TITLE
Update documentation to include argument parser

### LINKED ISSUE #166: Update Documentation to include argument Parser
We recently added argument parser for train and pretrain.py script i think we should update the documentation to reflect the same

### PR DESCRIPTION
**Pull request by Shashi456**
So i've included arguments in the documentation.
I was skeptical as to whether we should include the options for these arguments in the documentation or leave it to the user. 
I think the documentation should now look something like this. 
I will include the default value for these arguments.

Fixes #166.

### CODE REVIEW

**Comment by Shashi456** [comment]
@marco-c  please look at this

File: README.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ In the training phase, the best case is that we are able to detect between Y+D a
 
 The labeling technical details are described [in this issue](https://github.com/marco-c/autowebcompat/issues/2).
 
-### Training
+### Training methodologies
```
Comment (marco-c) [inline README.md]: Don't change this, it's unrelated to the PR.

File: README.md [hunk 2]
```diff
@@ -69,7 +69,21 @@ For the unsupervised training, we are using a related problem for which we alrea
 - Clone the repository with submodules: `git lfs clone --recurse-submodules REPO_URL`
 - Install the dependencies in requirements.txt: `pip install -r requirements.txt`.
 - Install the dependencies in test-requirements.txt: `pip install -r test-requirements.txt`.
-- Run the **pretrain.py / train.py** script to train the neural network.
+
+## Training the network
+- The **pretrain.py** or **train.py** script can be run to train the neural network.
+- There are options for running the same:
```
Comment (marco-c) [inline README.md]: Remove this line and add ", with the following options:" to the previous line.

File: README.md [hunk 3]
```diff
@@ -69,7 +69,21 @@ For the unsupervised training, we are using a related problem for which we alrea
...
+
+    ```
+    -network                  For training different kinds of
```
Comment (marco-c) [inline README.md]: The description for this should be "To select which network architecture to use".

File: README.md [hunk 4]
```diff
@@ -69,7 +69,21 @@ For the unsupervised training, we are using a related problem for which we alrea
...
+                              network
+                              
+    -optimizer                For training a network with different
```
Comment (marco-c) [inline README.md]: The description for this should be "To select the optimizer to use".

File: README.md (hunk 1 above)
Comment (Shashi456) [reply README.md]: I thought "training" would be redundant and so i changed it . I'll revert it back

**Comment by Shashi456** [comment]
@marco-c  i was wondering if we should include supported architectures and optimizers?

**Comment by marco-c** [comment]
> @marco-c i was wondering if we should include supported architectures and optimizers?

No, I'm pretty sure if we do that they will get stale (as we will forget to update the README every time we add a new architecture or optimizer).

**Pull request merged**